src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenInParametersTests.cs

```diff
@@ -11,6 +11,493 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests
     [CompilerTrait(CompilerFeature.ReadOnlyReferences)]
     public class CodeGenInParametersTests : CompilingTestBase
     {
+        [Fact]
+        public void InParamCallOptionalArg()
+        {
+            var comp = CompileAndVerify(@"
+using System;
+class C
+{
+    public static void Main()
+    {
+        int x = 1;
+        M(in x);
+    }
+    static void M(in int x, int y = 0)
+    {
+        Console.WriteLine(x);
+        Console.WriteLine(y);
+    }
+}", expectedOutput: @"1
+0");
+            comp.VerifyIL("C.M", @"
```

Comment:
>C.M [](start = 27, length = 3)

Is IL of this method really interesting? It feels like we should be checking IL for "Main".